Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM php:7.0-apache
+
+RUN apt-get update \
```

Comment:
Most likely you would end up modifying it for production to include your application specific needs. The primary intent is to allow developers to start with a consistent development environment.